citesphere/src/main/webapp/WEB-INF/views/auth/group/item.jsp

```diff
@@ -250,129 +257,222 @@
-			var identifier = $(clicked).attr('data-identifier');
-			var identType = $(clicked).attr('data-identifier-type');
-			if (identType != undefined && identType != "") {
-				identifier = identifier + " (" + identType + ")";
-			}
-			$("#parsed-ref-ident").text(identifier);
-			$("#parsed-ref-first-page").text($(clicked).attr('data-first-page'));
-			$("#parsed-ref-last-page").text($(clicked).attr('data-last-page'));
-			$("#parsed-ref-volume").text($(clicked).attr('data-volume'));
-			$("#parsed-ref-source").text($(clicked).attr('data-source'));
-			$("#parsed-ref-publication-type").text($(clicked).attr('data-publication-type'));
-			$("#parsed-ref-reference-id").text($(clicked).attr('data-reference-id'));
-			$("#parsed-ref-reference-label").text($(clicked).attr('data-reference-label'));
-			$("#parsed-ref-citation-id").text($(clicked).attr('data-citation-id'));
-			$("#parsed-ref-ref-raw").text($(clicked).attr('data-ref-raw'));
-            
-			$("#parsed-ref-modal").modal();
-		});
-	})
+		$(".parse-ref-info").click(
+				function(e) {
+					var clicked = e.target;
+					$("#ref-unparsed").text($(clicked).text());
+					$("#parsed-ref-authors").text(
+							$(clicked).attr('data-authors'));
+					$("#parsed-ref-title").text($(clicked).attr('data-title'));
+					$("#parsed-ref-year").text($(clicked).attr('data-year'));
+					var identifier = $(clicked).attr('data-identifier');
+					var identType = $(clicked).attr('data-identifier-type');
+					if (identType != undefined && identType != "") {
+						identifier = identifier + " (" + identType + ")";
+					}
+					$("#parsed-ref-ident").text(identifier);
+					$("#parsed-ref-first-page").text(
+							$(clicked).attr('data-first-page'));
+					$("#parsed-ref-last-page").text(
+							$(clicked).attr('data-last-page'));
+					$("#parsed-ref-volume")
+							.text($(clicked).attr('data-volume'));
+					$("#parsed-ref-source")
+							.text($(clicked).attr('data-source'));
+					$("#parsed-ref-publication-type").text(
+							$(clicked).attr('data-publication-type'));
+					$("#parsed-ref-reference-id").text(
+							$(clicked).attr('data-reference-id'));
+					$("#parsed-ref-reference-label").text(
+							$(clicked).attr('data-reference-label'));
+					$("#parsed-ref-citation-id").text(
+							$(clicked).attr('data-citation-id'));
+					$("#parsed-ref-ref-raw").text(
+							$(clicked).attr('data-ref-raw'));
+
+					$("#parsed-ref-modal").modal();
+				});
+    
+		
+    $('#delete').click(function(e) {
+	    e.preventDefault();	 
+	    $('#deleteModal').modal("hide");
+	    $('#loadModal').modal("show");
+	    $.ajax({
+		    url : '<c:url value="/auth/group/${zoteroGroupId}/items/${citation.key}"/>'+ "?${_csrf.parameterName}=${_csrf.token}",
+		    method : 'DELETE',
+	        'success' : function(data) {
+				
+
+                $('#loadModal').modal("hide");
+                $.notify('<i class="fas fa-check-circle"></i> Citation successfully deleted!',
```

Comment:
the notification should stay there until the new page is loaded and it should say
"Citation successfully delete. You will be redirected to the citation list page."